processing/src/test/java/io/druid/segment/data/IncrementalIndexTest.java

```diff
@@ -436,6 +537,8 @@ public void run()
         );
       }
     }
+
+    index.close();
```

Comment:
i dint go that far becoz they are only working with IncrementalIndex and dint exist previously. That said, you are right and I will take care of this in #2325